core/clustersmngr/cluster/single.go

```diff
@@ -4,14 +4,17 @@ import (
 	"fmt"
 	"net"
 
+	//nolint:gci
```

Comment:
Better to run `golangci-lint run --fix` to avoid this suppression. I can add a new target to our makefile: `make lint-fix`.